app/views/trainees/new.html.erb

```diff
@@ -14,6 +14,8 @@
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <%= form_with model: @trainee, local: true do |f| %>
 
+      <%= f.govuk_error_summary %>
```

Comment:
The error summary is usually position before the main page heading, in this case around line 8 "Add a trainee". As `f.govuk_error_summary` can only be called inside a form, it means that the form itself will need to start further up the page. When you move this be sure to move the closing `<% end %>` so that the markup it produces is valid.

Reply:
Yeah I'd position as the first element after `main`

Reply:
It's not going to be easy to position it directly after `<main>` but its near enough that if the user were to use the skip link as part of normal keyboard navigation, they would still have visibility of it

```
<main class="govuk-main-wrapper " id="main-content" role="main">
  <%= yield :start_page_banner %>
  <div class="govuk-width-container">
    <%= yield %>
  </div>
</main>
```

Reply:
cheers Al!